modules/API/src/main/java/de/Keyle/MyPet/api/entity/MyPetType.java

```diff
@@ -35,6 +35,9 @@ public enum MyPetType {
     Allay("ALLAY", "1.19", MyAllay.class, new Compat<>()
             .v("1.19", "allay")
             .search()),
+    Armadillo("ARMADILLO", "1.21", MyArmadillo.class, new Compat<>()
```

Comment:
Weren't they added in 1.20.5/6?

Reply:
But this is fine too - once it's in dev I can easily just copy paste some stuff into v1_20_R4 and change that

Reply:
aight i put it in dev branch

Reply:
i thought it was from 1.21 lmfao